README.md

```diff
@@ -1,10 +1,43 @@
 # ereadingtool
-Build [![CircleCI](https://circleci.com/gh/ereadingtool/ereadingtool.svg?style=svg)](https://circleci.com/gh/ereadingtool/ereadingtool)
+The ereadingtool allows students and instructors to consume or provide a language's texts and interactively view its translations. Since languages rarely have a one-to-one mapping, texts created on the app can have their translation modified or changed. Students who consume the text material are presented questions created by an instructor for that particular text. Scores and progress is tracked and can be converted to a PDF for alternative viewing.
+
+## Overview
+Ereadingtool is a SPA delivered by an NGINX webserver. The SPA communicates with a Django API server using a REST-ful paradigm. The Django web framework utilizes websocket connections for smooth interaction between the client and the server. It is supported by a Redis in-memory database to quickly serve the texts to the client over this websocket connection. Note that this is a microservice architecture. 
```

Comment:
Might be good to expand on "micoservice architecture".  Maybe something like "Note that this is a microservice architecture, but deployed to a single server."